synapse/cores/lmdb.py

```diff
@@ -252,9 +349,9 @@ def _initDbConn(self):
         map_slack = self._link[1].get('lmdb:mapslack', 2 ** 30)
         self._map_slack, _ = s_datamodel.getTypeNorm('int', map_slack)
 
-        # Maximum number of 'databases', really tables.  We use 4 different tables (1 main plus
-        # 3 indices)
-        MAX_DBS = 4
+        # Maximum number of 'databases', really tables.  We use 5 different tables (1 main plus
+        # 3 indices and a blob store), + 10 tables for possible migration use cases.
+        MAX_DBS = 5 + 10
```

Comment:
You can change it in the future without any backwards compatibility issue. The +10 just take up (a little) extra memory.